certbot-nginx/certbot_nginx/tls_sni_01.py

```diff
@@ -100,9 +100,13 @@ def _mod_config(self, ll_addrs):
             if line[0] == ['http']:
                 body = line[1]
                 found_bucket = False
+                posn = 0
                 for inner_line in body:
                     if inner_line[0] == bucket_directive[1]:
+                        if int(inner_line[1]) < int(bucket_directive[3]):
+                            body[posn] = [bucket_directive[1], bucket_directive[3]]
```

Comment:
Thanks for catching this. I was going off of what `inner_line` looked like when I printed it in ipdb, not what it actually was, which is kind of surprising because only a few lines below is `body.insert(0, bucket_directive)`.